bin.src/makeCoaddTempExp.py

```diff
@@ -20,9 +20,5 @@
 # the GNU General Public License along with this program.  If not,
 # see <http://www.lsstcorp.org/LegalNotices/>.
 #
-import lsst.pex.logging as pexLog
 from lsst.pipe.tasks.makeCoaddTempExp import MakeCoaddTempExpTask
-
-pexLog.Trace.setVerbosity('lsst.coadd', 3)
-pexLog.Trace.setVerbosity('lsst.ip.diffim', 1)
```

Comment:
Agree.  I'll keep this commit, with those verbosity setting removed. 

If coadd and ip_diffim need to be more verbose, we probably should change the logging records to a higher level in coadd and ip_diffim.  (I can file a ticket if that is wanted)